### ISSUE TITLE
iphone 5s plus interface issue

### ISSUE THREAD

**Issue by amiresfahani** [pomozoff/caspianphone#101]
check image please

![3cc100d5b97f620b6ee47ad3513034f9565a1157c718d61ca220b68e4db1399f](https://cloud.githubusercontent.com/assets/8526958/5721979/252630c4-9b4d-11e4-9d00-1f46462a9739.jpg)

**Comment by pomozoff** [comment]
Yes, I see these kind of bugs a lot of times. The reason of it is that linphone for iOS uses very old legacy unsupported code and they do not want to refactor it to use new iOS SDK features like an Auto Layout. But old iOS sdk was not made for devices with different screen sizes. Old sdk does not consider a new iOS paradigm give whole screen to the app, linphone's sources consider the status bar presence and tries to fix output by fixing frames by pixels. This is bad in the current reality and prohibited by Apple's guidelines.
I think I spend too much time debugging an ugly linphone's sources to fix these tiny bugs. I afraid to implement new features because I have no tests and any change may break something again and I need to fix at.
I think we need to make an iOS library to wrap linphone's C-API and provide to an iOS app simple but reach Cocoa API. Then we may build our app without looking at linphone-iphone project and use every feature of the current sdk. Also this split to library and UI parts facilitates the whole app development and support it, we can develop both parts in parallel.

**Comment by Gui13** [comment]
Hey guys,
Just a heads-up, we fixed this issue a month ago.
Indeed this was apple dropping legacy support for rotation or Windows.

If you merge this will be fixed.

**Comment by pomozoff** [comment]
Hi, I frequently sync our project with your master branch.
Last sync was a few days ago and I did not see this glitch on my iPhone 5.
I think this is an app from AppStore.

**Comment by amiresfahani** [comment]
yes it is the app store version
